src/TesApi.Web/BatchScheduler.cs

```diff
@@ -1094,18 +1147,28 @@ private async Task<BatchModels.StartTask> GetStartTaskAsync(string poolId, Batch
                 cmd.Append($" && {CreateWgetDownloadCommand(await UploadScriptAsync(script, new(await ReadScript(vmFamilyStartupScript))), $"{BatchNodeTaskWorkingDirEnvVar}/{script}", setExecutable: true)} && {BatchNodeTaskWorkingDirEnvVar}/{script}");
             }
 
+            switch (vmFamilySeries)
+            {
+                case VmFamilySeries.standardNFamily:
+                    extensions = extensions.Append(new(name: "gpu", publisher: "Microsoft.HpcCompute", type: "NvidiaGpuDriverLinux", autoUpgradeMinorVersion: true) { TypeHandlerVersion = "1.6" });
```

Comment:
should we pull this out as a const for easier future upgrades?

Reply:
It's not at all clear what part(s) of this, if any, would change over time. I added the link to the doc for this specific extension.